packages/node_modules/@node-red/editor-client/src/js/ui/clipboard.js

```diff
@@ -731,7 +731,7 @@ RED.clipboard = (function() {
                 nodes.unshift(parentNode);
                 nodes = RED.nodes.createExportableNodeSet(nodes);
             } else if (type === 'full') {
-                nodes = RED.nodes.createCompleteNodeSet(false);
```

Comment:
Passing a bare `false` is equivalent to `{credentials: false}` - but better to be explicit.